frontend/pages/admin/IntegrationsPage/cards/CertificateAuthorities/components/CustomSCEPForm/_styles.scss

```diff
@@ -0,0 +1,13 @@
+.custom-scep-form {
+  &__fields {
+    display: flex;
+    flex-direction: column;
+    gap: $pad-large;
+  }
+
+  &__cta {
```

Comment:
I think if you rename the class to `modal-cta-wrap`, you won't need to style this over and over